### PR TITLE
Document CorlexFeatureExtractor and add test class TestCorlex

### PR DESCRIPTION
**Pull request by yxd126**
Solving issue #100

### CODE REVIEW

File: ner/pom.xml [hunk 1]
```diff
@@ -89,6 +89,11 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.4</version>
         </dependency>
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-ner</artifactId>
+            <version>3.0.58</version>
+        </dependency>
```
Comment (danyaljj) [inline ner/pom.xml]: why are you adding this?

Comment (yxd126) [inline ner/pom.xml]: I didn't see this. I only committed 2 files, but it seems that I missed one. I think you can ignore this. Sorry for that.

Comment (danyaljj) [inline ner/pom.xml]: No worries. You can delete it now.

File: edison/src/main/java/edu/illinois/cs/cogcomp/edison/features/factory/CorlexFeatureExtractor.java [hunk 2]
```diff
@@ -80,9 +89,10 @@ private synchronized static void loadData() throws EdisonException {
         String lemma = WordHelpers.getLemma(ta, wordPosition);
 
         Set<Feature> features = new LinkedHashSet<>();
-        if (data.containsKey(lemma))
+        if (data.containsKey(lemma)) {
+            String tmp = data.get("give");
```
Comment (danyaljj) [inline edison/src/main/java/edu/illinois/cs/cogcomp/edison/features/factory/CorlexFeatureExtractor.java]: why this line?

File: edison/src/test/java/edu/illinois/cs/cogcomp/edison/features/lrec/TestCorlex.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                {"atr"};
+
+        if (feats == null) {
+            log.debug("Feats are returning NULL.");
+        }
+
+        log.debug("Printing Set of Features");
+        for (Feature f : feats) {
+            log.debug(f.getName());
+            assert (ArrayUtils.contains(expected_outputs, f.getName()));
```
Comment (danyaljj) [inline edison/src/test/java/edu/illinois/cs/cogcomp/edison/features/lrec/TestCorlex.java]: use `assertTrue` syntax. See other tests.

**Comment by yxd126** [comment]
Corrected all the mistakes

**Comment by danyaljj** [comment]
Looks great! Thanks!

**Pull request merged**